features/tags_and_wrangling/tag_wrangling.feature

```diff
@@ -227,3 +227,32 @@ Feature: Tag wrangling
     When I edit the tag "Cabin Pressure"
     Then I should not see "Sign Up"
       And I should see the tag wrangler listed as an editor of the tag
+
+  Scenario: A user can not see the reindex button on a tag page
+
+    Given the following typed tags exists
+        | name              | type         | canonical |
+        | Cowboy Bebop      | Fandom       | true      |
+      And I am logged in as a random user
+      When I view the tag "Cowboy Bebop"
```

Comment:
Your Whens and Thens are indented too much in these tests -- they need to line up with the Givens, not the Ands.